packages/nextjs/components/Header.tsx

```diff
@@ -8,12 +8,10 @@ import { FaucetButton, RainbowKitCustomConnectButton } from "~~/components/scaff
  */
 export const Header = () => {
   return (
-    <div className="sticky lg:static top-0 navbar bg-base-100 min-h-0 flex-shrink-0 z-20 shadow-md shadow-secondary px-0 sm:px-2">
-      <div className="md:navbar-end text-right items-center lg:w-1/2">
+    <div className="sticky lg:static top-0 navbar bg-base-100 min-h-0 flex-shrink-0 z-20 mb-2 shadow-md shadow-secondary px-0 sm:px-2">
```

Comment:
With the extra margin it makes it look like the navbar elements are not centered vertically. How about we remove the shadow completely? Remove the following properties: `mb-2 shadow-md shadow-secondary`. I played around with changing the color of the shadow but I didn't like the result.

Reply:
> With the extra margin it makes it look like the navbar elements are not centered vertically. How about we remove the shadow completely? Remove the following properties: `mb-2 shadow-md shadow-secondary`. I played around with changing the color of the shadow but I didn't like the result.

done.